src/main/java/edu/oakland/helper/admin/LocationDataPoint.java

```diff
@@ -69,6 +69,10 @@ public LocalDateTime getTime() {
     return time;
   }
 
+  public String toString() {
+    return lat + " " + lng + " " + time;
```

Comment:
I would see if you can make this more descriptive, may something like:
```suggestion
    return "Location Data at " + time.toString() + ":\n\tLatitude: " + lat + "\n\tLongitude: " + lng;
```